app/models/miq_shortcut.rb

```diff
@@ -37,7 +37,7 @@ def self.seed
   def self.fixture_file_names
     shortcuts_dir = FIXTURE_DIR.join("miq_shortcuts")
     main_shortcuts = FIXTURE_DIR.join("miq_shortcuts.yml")
-    [main_shortcuts, (shortcuts_dir.directory? ? shortcuts_dir : nil)]
+    [main_shortcuts, (shortcuts_dir if shortcuts_dir.directory?)]
```

Comment:
again, not sure about these embedded trailing `if`s

Reply:
this could be simplified. but is no longer a `nil` case - I'll pull out

Reply:
ok, I put into its own commit. this is overly complex